Configuration/AlCa/python/autoAlca.py

```diff
@@ -131,4 +131,5 @@ def buildList(pdList, matrix):
     'PromptCalibProdSiPixelAli',       # AlignmentProducerAsAnalyzer, MillePedeFileConverter
     'PromptCalibProdBeamSpotHP',       # AlcaBeamSpotProducer
     'PromptCalibProdBeamSpotHPLowPU',  # AlcaBeamSpotProducer
+    'PromptCalibProdLumiPCC',          # AlCaPCCRandom producer
```

Comment:
@tvami , if here you add 'AlCaPCCRandom', I think it will work. The failure is already at step2 which runs AlCaPCCRandom. I tried it locally, could reproduce the error and after adding this producer to autoAlCa.py, it ran without problems.

Reply:
Thanks @malbouis !